app/assets/stylesheets/landing_page/mixins/_transitions.scss

```diff
@@ -0,0 +1,19 @@
+/*
+ * This is the transition that should be used with link and button hovers.
+ *
+ * Usage:
+ *
+ * .my-button:hover {
+ *   @include hover-transition
+ * }
+ *
+ * To specify which property should be animated, use:
+ *
+ * .my-button:hover {
+ *   @include hover-transition(background-color)
+ * }
+ *
+ */
+@mixin hover-transition($transition-prop: all) {
```

Comment:
Transitioning all properties can be bad for performance, so keep an eye on this.

If you need to explicitly list multiple transitions without using `all`, this mixin wouldn't work anymore since they have to be comma separated.